Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -1811,8 +1811,28 @@ def get_mapping_fields_command():
     return mapping_response
 
 
+def get_modified_remote_data_command(client, args):
+    demisto.debug('Performing get-modified-remote-data command')
+
+    remote_args = GetModifiedRemoteDataArgs(args)
+    last_update = remote_args.last_update  # In the first run, this value will be set to 1 minute earlier
+    last_update_utc = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})  # convert to utc format
+    last_update_without_ms = last_update_utc.isoformat().split('.')[0]
+
+    raw_incidents = client.get_incidents(gte_modification_time=last_update_without_ms, limit=100)
+
+    modified_incident_ids = list()
+    for raw_incident in raw_incidents:
+        incident_id = raw_incident.get('incident_id')
+        modified_incident_ids.append(incident_id)
+
+    return GetModifiedRemoteDataResponse(modified_incident_ids)
```

Comment:
same here. before returning, print to debug the modified_incident_ids

Reply:
I'm actually printing the returned incidents in CSP like the GetRemoteDataResponse class